airflow/models/dagbundle.py

```diff
@@ -39,5 +39,7 @@ class DagBundleModel(Base):
     latest_version = Column(String(200), nullable=True)
     last_refreshed = Column(UtcDateTime, nullable=True)
 
-    def __init__(self, *, name: str):
+    def __init__(self, *, name: str, latest_version=None, last_refreshed=None):
         self.name = name
+        self.latest_version = latest_version
```

Comment:
Did you end up using these somewhere? I'd intentionally left them off, as specifying them during construction didn't feel useful.

Reply:
i don't think so; i think i previously did.